src/components/study-container.jsx

```diff
@@ -677,10 +675,7 @@ export function StudyContainer({ view, onChangeTab }) {
     );
 
     useEffect(() => {
-        if (
-            (studyUuid && currentRootNetworkUuid && !isStudyNetworkFound) ||
-            (currentRootNetworkRef.current && currentRootNetworkRef.current !== currentRootNetworkUuid)
-        ) {
```

Comment:
this is an error.
If `currentRootNetworkUuid` change, you must check network existence, check study indexation for this root network etc... then you must keep this

Reply:
OK! I reverted while keeping condition with previous value in ref